pkgs/by-name/im/imgui/package.nix

```diff
@@ -84,15 +86,21 @@ stdenv.mkDerivation rec {
     owner = "ocornut";
     repo = "imgui";
     tag = "v${version}";
-    hash = "sha256-6j4keBOAzbBDsV0+R4zTNlsltxz2dJDGI43UIrHXDNM=";
+    hash = "sha256-J+h7jJ+4wqr6RivtzyTDMXKxFoGs7dQbzqdu51XgEbc=";
   };
 
   cmakeRules = "${vcpkgSource}/ports/imgui";
   postPatch = ''
     cp "$cmakeRules"/{CMakeLists.txt,*.cmake.in} ./
   '';
 
-  nativeBuildInputs = [ cmake ];
+  nativeBuildInputs = [
+    cmake
+    xorg.libX11
+    xorg.libXcursor
+    xorg.libXrandr
+    wayland
+  ];
```

Comment:
> For some reason it failed to build glfw without adding as native deps xorg and wayland, this was not needed before and I have no idea why.

Huh, `native` specifically, not `buildInputs`? For wayland it could be `wayland-scanner` maybe, but the rest are supposed to be... just libraries?

Could I ask you to to publish the build logs for the failing version please? A gist e.g.